spec/integration/outputs/update_spec.rb

```diff
@@ -9,7 +9,8 @@ def get_es_output( options={} )
       "index" => "logstash-update",
       "template_overwrite" => true,
       "hosts" => get_host_port(),
-      "action" => "update"
+      "action" => "update",
+      "script_lang" => "groovy"
```

Comment:
So we're using a groovy script here, because it works with older versions of Elasticsearch too?  It'd be nice to get a "painless" script eventually, I think, but it's not a deal breaker now.